package.json

```diff
@@ -41,15 +41,17 @@
   "homepage": "https://github.com/SleeplessByte/triggerbus#readme",
   "devDependencies": {
     "@types/node": "^11.13.0",
+    "@typescript-eslint/eslint-plugin-tslint": "^1.6.0",
```

Comment:
We don't want this one, see above.

MIssing the import and unicorn plugin dependency here. Might not need them at all.